frontend/projects/upgrade/src/styles.scss

```diff
@@ -236,9 +236,10 @@
   width: 805px;
 }
 
-.mat-button,
-.mat-flat-button,
-.mat-stroked-button {
+.mat-mdc-button,
+.mat-mdc-unelevated-button,
+.mat-mdc-outlined-button {
+  /* TODO(mdc-migration): The following rule targets internal classes of button that may no longer apply for the MDC version.*/
   &.cdk-program-focused .mat-button-focus-overlay {
```

Comment:
I'm not sure what this was even doing? May be able to remove?

Reply:
I removed them as these classes don't seem to be used.